### DISCUSSION TITLE
headings autoformatting hashes (##)

### DISCUSSION THREAD

**Discussion by matbad** [TriliumNext/Trilium#2992] [Q&A] [answered]
When using headings in text notes I can see, that my text ist autoformatted in size. But the hashes are still there.

In the example shown in https://github.com/zadam/trilium/wiki/Markdown I can see, that the hashes are not there, when the Markdown text is pasted from the clipboard.

How can i get the formatting, like in the example linked?

Thanks for your help.

**Comment by sigaloid** [comment, marked as answer]
Open options (top left logo -> options) and under appearance, change heading style to underline

**Comment by matbad** [comment]
Wow, i don't know how i was able to miss that. Thank you.
